cli/flox/doc/manifest.toml.md

```diff
@@ -309,13 +309,18 @@ It will be removed in a later release.
 
 ## `[profile]`
 
-Scripts defined in the `[profile]` section are sourced by *your shell* and
-inherit environment variables set in the `[vars]` section and by the `[hook]`
-scripts.
-The `profile.common` script is sourced for every shell,
-and special care should be taken to ensure compatibility with all shells.
-The `profile.<shell>` scripts are then sourced *after* `profile.common` by the
-corresponding shell.
+Scripts defined in the `[profile]` section
+are sourced by *your shell*
+and inherit environment variables
+set in the `[vars]` section
+and by the `[hook]` scripts.
+The `profile.common` script is sourced
+for every shell
+and special care should be taken
+to ensure compatibility with all shells.
+The `profile.<shell>` scripts
+are then sourced *after* `profile.common`
+by the corresponding shell.
```

Comment:
Reverting this change as it was an unintended rebase artifact and does not affect the content 
```suggestion
Scripts defined in the `[profile]` section are sourced by *your shell* and
inherit environment variables set in the `[vars]` section and by the `[hook]`
scripts.
The `profile.common` script is sourced for every shell,
and special care should be taken to ensure compatibility with all shells.
The `profile.<shell>` scripts are then sourced *after* `profile.common` by the
corresponding shell.
```